doc/nrf/ug_nrf5340.rst

```diff
@@ -106,165 +85,246 @@ Zephyr includes the `OpenAMP`_ library, which provides a complete solution for e
-+-----+--------------------------+-------------------------------+--------------------------------------------+
-|`d)` |``n``                     |``n``                          |some other possibilities, not defined here  |
-+-----+--------------------------+-------------------------------+--------------------------------------------+
+The combination of these options determines which (if any) sample is included in the build for the application sample as a child image for the ``nrf5340_dk_nrf5340_cpunet`` core:
 
-`a)` This option is used for all Bluetooth Low Energy samples in the |NCS| since in this case :option:`CONFIG_BT_RPMSG_NRF53` defaults to ``y``.
++---------------------------------+--------------------------------------+--------------------------------------------+
+| :option:`CONFIG_BT_RPMSG_NRF53` | :option:`CONFIG_NRF_802154_SER_HOST` | Child image sample for the network core    |
++=================================+======================================+============================================+
+|``y``                            |``n``                                 |:ref:`zephyr:bluetooth-hci-rpmsg-sample`    |
++---------------------------------+--------------------------------------+--------------------------------------------+
+|``n``                            |``y``                                 |:ref:`zephyr:nrf-ieee802154-rpmsg-sample`   |
++---------------------------------+--------------------------------------+--------------------------------------------+
+|``y``                            |``y``                                 |:ref:`multiprotocol-rpmsg-sample`           |
++---------------------------------+--------------------------------------+--------------------------------------------+
+|``n``                            |``n``                                 |some other possibilities, not defined here  |
++---------------------------------+--------------------------------------+--------------------------------------------+
 
-`b)` This option is used by default for Thread and Zigbee samples in the |NCS| since in this case :option:`CONFIG_NRF_802154_SER_HOST` defaults to ``y``.
 
-`c)` This option is used for Thread and Zigbee samples in the |NCS| when the Bluetooth LE is additionally enabled by the user.
+Configuration of the child image
+================================
 
-SES is unable to automatically program the network sample :ref:`zephyr:bluetooth-hci-rpmsg-sample` to the network core when it is added as a child image.
-To program the network sample to the network core, see `Programming the network sample from SES`_.
+When a network sample is built automatically as a child image in a multi-image build, you can define the relevant Kconfig options (if required) in a :file:`.conf` file.
+Name the file *network_sample*\ .conf, where *network_sample* is the name of the child image (for example, ``hci_rpmsg.conf``).
+Place the file in a :file:`child_image` subfolder of the application sample directory.
```

Comment:
Please point to the description in the `multi_image.rst` which explains this as there are several alternatives on how to do this.

Reply:
https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_multi_image.html#image-specific-variables